OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -301,14 +301,14 @@ protected virtual void Tick(Actor self)
 
 			// Add land activity if LandOnCondidion resolves to true and the actor can land at the current location.
 			if (!ForceLanding && landNow.HasValue && landNow.Value && airborne && CanLand(self.Location)
-				&& !(self.CurrentActivity is HeliLand || self.CurrentActivity is Turn))
+				&& !((self.CurrentActivity is Land && Info.VTOL) || self.CurrentActivity is Turn))
```

Comment:
Does this have to be exclusive to VTOL craft?

Reply:
It doesn't have to, but might not work well with non-VTOLs. I removed the limitation, but fixing any bugs with non-VTOLs would likely require refactoring `Land` further to handle the case of "already at target location, but up in the air" for non-VTOLs, which I feel is a bit out of scope for this PR.